Contest_API.md

```diff
@@ -671,13 +674,22 @@ Returned data:
 ```json
 [{
    "id": "java",
-   "name": "Java"
+   "name": "Java",
+   "require_entry_point": true,
+   "entry_point_description": "Main class",
+   "extensions": ["java"]
 }, {
    "id": "cpp",
-   "name": "GNU C++"
+   "name": "GNU C++",
+   "require_entry_point": false,
+   "entry_point_description": null,
```

Comment:
I would prefer leaving the `null`ed entry out from the examples.

Reply:
Ditto